contrib/perf/test-full-workflow.sh

```
@@ -0,0 +1,147 @@
+echo " Testing Performance Report Workflow"
+echo "======================================="
+echo "Run ID: $RUN_ID"
+echo ""
+
+# Clean up previous test runs
+rm -rf test-workflow
+mkdir -p test-workflow
+cd test-workflow
+
+echo "- Step 1: Getting PR information..."
+PR_INFO=$(python ../get-pr-info.py --run-id $RUN_ID --format json)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)